Mezon/Fs/Layer.php

```diff
@@ -46,6 +46,30 @@ public static function filePutContents(string $filePath, string $data, int $flag
      */
     private static $createdDirectories = [];
 
+    /**
+     * Checking if the directory exists
+     *
+     * @param string $dirPath
+     *            path to the checking directory
+     * @return bool true if the directory exists, false otherwise
+     */
+    public static function directoryExists(string $dirPath): bool
+    {
+        if (Conf::getConfigValueAsString('fs/layer', 'real') === 'real') {
+            // @codeCoverageIgnoreStart
+            return file_exists($dirPath);
+            // @codeCoverageIgnoreEnd
+        } else {
+            self::$createdDirectories = array_reverse(self::$createdDirectories);
+
+            $result = array_pop(self::$createdDirectories);
+
+            self::$createdDirectories = array_reverse(self::$createdDirectories);
+
+            return $result;
```

Comment:
This will not work. Look at the line #91 in this file. And you shall see that the array self::$createdDirectory is filled with 
```php
[
                'path' => $filePath,
                'mode' => $mode,
                'recursive' => $recursive
            ];
```
So to find out that the directory was created you need to iterate through `self::$createdDirectories` and check if the element of this array `$item['path']` have value $dirPath